docs/tutorials/kube-ingress-aws.md

```diff
@@ -197,42 +198,6 @@ In the above example we create a default path that works for any hostname, and
 make use of the `external-dns.alpha.kubernetes.io/hostname` annotation to create
 multiple aliases for the resulting ALB.
 
-## Dualstack ALBs
-
-AWS [supports](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/application-load-balancers.html#ip-address-type) both IPv4 and "dualstack" (both IPv4 and IPv6) interfaces for ALBs.
-The Kubernetes Ingress AWS controller supports the `alb.ingress.kubernetes.io/ip-address-type`
-annotation (which defaults to `ipv4`) to determine this. If this annotation is
-set to `dualstack` then ExternalDNS will create two alias records (one A record
-and one AAAA record) for each hostname associated with the Ingress object.
-
-Example:
-
-```yaml
```

Comment:
what happens to this example?

Reply:
Since this change creates both A & AAAA records (when ALIAS records are enabled) regardless of the loadbalancer being dual stack, this example isn't really relevant to external DNS anymore.

Reply:
https://github.com/kubernetes-sigs/external-dns/pull/5111#discussion_r1968302145